lib/board.rb

```diff
@@ -0,0 +1,52 @@
+class Board
+  attr_accessor :cells
+
+  def initialize
+    reset!
+  end
+
+  # Resets the state of the cells in the board to a 9 element array of " ".
+  def reset!
+    @cells = Array.new(9, " ")
```

Comment:
Try to access your instance variables using your settors/gettors via attr_accessor. 

`self.cells = Array.new(9, " ")`